mozetl/schemas.py

```diff
@@ -0,0 +1,10 @@
+import ujson as json
+import os
+
+from pyspark.sql.types import StructType
+
+directory = os.path.join(os.path.dirname(__file__), 'json')
```

Comment:
This might not work when packaged as a binary. It might be worthwhile to pull the logic from [`topline.schema.schema_from_json`](https://github.com/mozilla/python_mozetl/blob/master/mozetl/topline/schema.py) into the utility library and use it here. It uses `pkg_resource` to ensure that it references the json file correctly.